### DISCUSSION TITLE
Scrum Prep Milestone 2-4

### DISCUSSION THREAD

**Discussion by NatiPerez459** [uprm-inso4101-2024-2025-s1/semester-project-mercado-colegial-online#141] [General]
Each team member will comment progress made through the past week.

**Comment by ActuallySeb** [comment]
Docker Implementation taking longer than expected

**Comment by PerezMarcelo** [comment]
Worked on Survey task to gain insight and feedback on future website users, sellers and clients.

**Comment by YampierFernandez** [comment]
Participated in meeting between backend and database team leads for planning, and created two research tasks.

**Comment by JadStelar** [comment]
Did some additional research and learning for future task implementation.

**Comment by AlvinBadillo** [comment]
Analyzed the current state of the code and found missing database entity components. Created tasks for missing database controllers and models that need to be implemented.

**Comment by JuanDelPueblo** [comment]
Worked on a documentation task to revise section 2.1.6.

**Comment by elliot-gomez** [comment]
This week, I completed revising sections 1.2.2, 1.3, 1.4, 1.5, and 2.1.4 of the Milestone 1 documentation. Each section was updated to address the professor's specific feedback, ensuring that the revisions aligned with the expectations and requirements provided.
- Elliot J. Gomez Coriano

**Comment by CovasJJ** [comment]
Focused on research to prepare for upcoming tasks.

**Comment by RyanVelez101** [comment]
focused on the survey task and was able to attend mercado colegial to collect data from possible users
-Ryan R. Velez Villarrubia

**Comment by cristian-barreras** [comment]
I was working on the documentation and its revision.

**Comment by FabiolaZTorres** [comment]
I worked on the Survey task by creating the form with the agreed-upon questions and assisted in surveying buyers and sellers at the Mercado Colegial.

- Fabiola Z. Torres Maldondo

**Comment by Christian2101** [comment]
Continued revising and fixing the documentation using the professors feedback

**Comment by gcal77** [comment]
I couldn't work too much on the project this week due to having multiple exams in these past days. However, I'm still working on implementing lazy image functionality into the home page.
